src/NATS.Client.Core/NatsConnection.cs

```diff
@@ -471,7 +471,7 @@ private async void ReconnectLoop()
             // If dispose this client, WaitForClosed throws OperationCanceledException so stop reconnect-loop correctly.
             await _socket!.WaitForClosed.ConfigureAwait(false);
 
-            _logger.LogTrace($"Detect connection {_name} closed, start to cleanup current connection and start to reconnect.");
+            _logger.LogTrace(NatsLogEvents.Connection, "Detect connection {Name} closed, start to cleanup current connection and start to reconnect", _name);
```

Comment:
Inconsistent tense with itself and other logs.

Reply:
Fixed as `Connection {Name} is closed. Will cleanup and reconnect`.
happy to hear other suggestions.